bindata/cloud-network-config-controller/managed/controller.yaml

```
@@ -62,6 +62,9 @@ spec:
                 topologyKey: kubernetes.io/hostname
       priorityClassName: hypershift-control-plane
       securityContext:
+{{- if not (eq .RunAsUser "")}}
+        runAsUser: {{.RunAsUser}}
+{{ - end }}
```

This is an invalid syntax, it should be:
```suggestion
{{- end }}
```